pkg/registration/register/common.go

```diff
@@ -196,6 +200,15 @@ func (a *AggregatedHubDriver) CreatePermissions(ctx context.Context, cluster *cl
 
 }
 
+func (a *AggregatedHubDriver) Accept(cluster *clusterv1.ManagedCluster) bool {
```

Comment:
nit, add a comment here on it will need each driver to accept.

Reply:
Validated that the Qiu Jian's comment has been addressed.